.github/workflows/analyze-and-build.yml

```diff
@@ -35,11 +35,16 @@ jobs:
         with: {ruby-version: ^2.6}
       - uses: actions/checkout@master
       - run: yarn install --frozen-lockfile
-      - run: gem install bundler:1.17.2 && bundle check || bundle install --frozen --path vendor/bundle
+      - run: gem install bundler:1.17.2
+      - run: bundle check || bundle install --frozen --path vendor/bundle
       - name: Raise the fs.inotify ulimits to 524288 watches/queued events/user instances
-        run: echo 524288 | sudo tee -a /proc/sys/fs/inotify/max_user_watches && echo 524288 | sudo tee -a /proc/sys/fs/inotify/max_queued_events && echo 524288 | sudo tee -a /proc/sys/fs/inotify/max_user_instances && sudo sysctl -p
-      - run: cd android && ./gradlew androidDependencies --console=plain
+        run: |
+          echo 524288 | sudo tee -a /proc/sys/fs/inotify/max_user_watches
+          echo 524288 | sudo tee -a /proc/sys/fs/inotify/max_queued_events
+          echo 524288 | sudo tee -a /proc/sys/fs/inotify/max_user_instances
+          sudo sysctl -p
       - run: echo 'org.gradle.workers.max=2' >> ./android/gradle.properties
+      - run: cd android && ./gradlew androidDependencies --console=plain
```

Comment:
Is there any particular reason that `--console=plain` is needed?

We do [have other options](https://docs.gradle.org/current/userguide/command_line_interface.html#sec:command_line_logging), though I don't really have a strong reason to move towards them.

Reply:
I could do the `-D` version in the config file, but I like their rich terminal output locally; it's just needed here so that Actions can show it nicely, if I recall correctly. This just seemed like the most obvious place to put it so that it would only apply to Actions, I suppose.